<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="45"
           failures="8"
           errors="0"
           skipped="0"
           time="1633.722">
  <properties />
            <testcase name="C2_through_sip" time="22.600" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期3001会Ring 【预期：2 实际：4】"
              >
<![CDATA[java.lang.AssertionError: 预期3001会Ring 【预期：2 实际：4】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:425)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C2_through_sip(Callback.java:237)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="D8_des_fxs" time="22.614" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期2002会Ring 【预期：2 实际：4】"
              >
<![CDATA[java.lang.AssertionError: 预期2002会Ring 【预期：2 实际：4】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:425)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.D8_des_fxs(Callback.java:476)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="C1_through_sps" time="22.077" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期3001会被挂断 【预期：4 实际：3】"
              >
<![CDATA[java.lang.AssertionError: 预期3001会被挂断 【预期：4 实际：3】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:425)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C1_through_sps(Callback.java:222)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="C7_through_e1" time="37.002" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="CDR源中继检测 【预期：null 实际：SPS1】"
              >
<![CDATA[java.lang.AssertionError: CDR源中继检测 【预期：null 实际：SPS1】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:266)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:252)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C7_through_e1(Callback.java:334)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="C3_through_iax" time="22.754" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期3001会Ring 【预期：2 实际：4】"
              >
<![CDATA[java.lang.AssertionError: 预期3001会Ring 【预期：2 实际：4】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:425)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C3_through_iax(Callback.java:254)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="C8_through_gsm" time="31.990" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="CDR呼叫方检测 【预期：18506938857 [callback18506938857] 实际：callback2000】"
              >
<![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：18506938857 [callback18506938857] 实际：callback2000】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:262)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:252)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C8_through_gsm(Callback.java:354)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="C4_through_spx" time="21.671" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期3001会被挂断 【预期：4 实际：3】"
              >
<![CDATA[java.lang.AssertionError: 预期3001会被挂断 【预期：4 实际：3】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:425)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.C4_through_spx(Callback.java:270)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="D5_des_queue" time="42.649" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="CDR被叫方检测 【预期：1100 [6700(1100)] 实际：6700(1100)】"
              >
<![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1100 [6700(1100)] 实际：6700(1100)】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:263)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:252)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback.D5_des_queue(Callback.java:429)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="B2_add_InRoute_destination" time="31.041" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.224" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.436" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D4_des_ringgroup" time="48.044" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D2_des_extension" time="48.335" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.117" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A2_add_trunks" time="134.860" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E1_sps" time="40.827" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A1_Init" time="21.969" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.300" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E4_bri" time="39.512" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E3_fxo" time="42.712" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E5_e1" time="0.000" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D1_des_hangup" time="7.147" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="9.174" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D3_des_ivr" time="48.463" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E2_spx" time="39.338" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.138" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B1_add_InRoute_trunk" time="237.189" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D6_des_conference" time="39.516" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="C6_through_bri" time="47.515" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.265" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="G2_recovery_inbound" time="47.843" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.374" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.168" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.145" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B3_apply" time="8.103" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="D7_des_disa" time="50.045" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="G1_recovery_callback" time="39.190" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.186" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="C5_through_fxo" time="45.831" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.156" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="A3_add_destination" time="14.560" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="F_delete" time="65.722" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="E6_gsm" time="0.001" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="30.862" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
                <testcase name="B2_add_InRoute_destination" time="31.057" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.Callback" />
      
</testsuite>
